src/arsenic/connection.py

```diff
@@ -4,17 +4,17 @@
 from functools import wraps
 from io import BytesIO
 from json import JSONDecodeError
+from logging import getLogger
 from pathlib import Path
 from typing import Any, Tuple
 from urllib.parse import urlparse, urlunparse
 from zipfile import ZIP_DEFLATED, ZipFile
 
 from aiohttp import ClientSession
-from structlog import get_logger
 
 from arsenic import errors, constants
 
-log = get_logger()
+log = getLogger(__name__)
```

Comment:
@ojii what do you think:
* __name__ for fine-grained logging, or
* __package__ for coarse logging?